client/ayon_core/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,157 @@
+from maya import cmds
+
+from ayon_core.hosts.maya.api import lib, plugin
+
+from ayon_core.lib import (
+    BoolDef,
+    NumberDef,
+)
+from ayon_core.pipeline import CreatedInstance
+
+
+def _get_animation_attr_defs(cls):
+    """Get Animation generic definitions."""
+    defs = lib.collect_animation_defs()
+    defs.extend(
+        [
+            BoolDef("farm", label="Submit to Farm"),
+            NumberDef("priority", label="Farm job Priority", default=50),
+            BoolDef("refresh", label="Refresh viewport during export"),
+            BoolDef(
+                "includeParentHierarchy", label="Include Parent Hierarchy"
+            ),
+            BoolDef(
+                "includeUserDefinedAttributes",
+                label="Include User Defined Attributes"
+            ),
```

Comment:
Can we preserve the tooltips for these attribute definitions?

E.g. we're currently losing [this](https://github.com/ynput/ayon-core/pull/336/files#diff-34154b3e70a9a31f619f8fe214e6eccec7234359cc015fce4181124c3190d279L48).